nengo_loihi/neurongroup.py

```diff
@@ -0,0 +1,64 @@
+from nengo_loihi.axons import AxonGroup
```

Comment:
I think I would call this `neurons.py`, and rename the current `neurons.py` to `neurontypes.py` ... or just put them in the same file, it wouldn't be that long of a file, and I don't think it's that confusing for them both to be in there.

Reply:
My problem with putting them together is that one is for nengo API neuron_types, and the other is for the nengo_loihi API. I'd prefer to keep those separate.

I would vote for a `neurontypes.py` for what is now `neurons.py` (is that better than `neuron_types.py`? is there precedent?). The main downside is that this now differs from core nengo, where the neuron types are in `neurons.py`.